src/components/createbook.js

```diff
@@ -0,0 +1,17 @@
+export default function CreateBook() {
+  return (
+    <>
+      <h1>Create Book </h1>
+      <div>
+        <input type="text" placeholder="Enter book name" />
+        <select name="auther">
+          <option value="volvo">select auther</option>
+          <option value="saab">Saab</option>
+          <option value="mercedes">Mercedes</option>
+          <option value="audi">Audi</option>
+        </select>
+        <button type="button">Add Book</button>
```

Comment:
- [ ] The button type should be a submit since the future purpose of this button would be to submit the names of the `book & author` 👍